### DISCUSSION TITLE
Error: Can't find the installer!

### DISCUSSION THREAD

**Discussion by johnleaselock** [grahampugh/erase-install#190] [Q&A]
I am new to this script and am testing it out on some test systems.  I am trying to upgrade to Monterey (12.1) from both BigSur (11.6.2) and Catalina (10.15.7) using erase-install.sh version 25.0.  My first attempts were thru Jamf Pro (10.35.0) and when that was not working by manually kicking the process off via a terminal command.

I have tried multiple options for this script:
- */erase-install.sh --force-curl --reinstall --check-power --depnotify --overwrite --build=21C52
- */erase-install.sh --force-curl --update --reinstall --check-power --depnotify
- */erase-install.sh --force-curl --build=21A559 --reinstall --check-power
- */erase-install.sh --force-curl --erase --check-power --confirm

Both systems fail after creating and mounting the sparseimage with the same error **"[erase-install] ERROR: Can't find the installer!"** and the script quits.  

I have restarted both systems multiple times and reach time have removed the previously downloaded Install* file before starting the test again

Thank you for your help,
John
[erase-install.log](https://github.com/grahampugh/erase-install/files/7873989/erase-install.log)
.

**Comment by sospon** [comment]
Was having a similar issue that after the download of the .sparseimage, it did not start itself.
Try with adding the "--pkg" flag to download a different format of the installer and check if it will work that way.

**Reply by johnleaselock** [reply]
I will try this and let you know how it goes.
Thank you
